bin/ds

```diff
@@ -485,6 +485,9 @@ function pull_db {
       echo "Reindexing Solr..."
       drush solr-mark-all && drush solr-index
 
+      echo "Disabling Optimizely module on staging."
+      drush dis optimizely
```

Comment:
this should maybe be done earlier in the process, perhaps before cache is cleared?